Reword post-apply stage to remove ambiguity (#723)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/states.mdx

## The Post-Plan Stage

> The post-plan phase executes any configured run tasks after the plan is successfully complete, so it only occurs if there are [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) enabled

Reviewer: ```suggestion
The post-plan phase executes any configured run tasks after the plan is successful, so it only occurs if there are [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) enabled
```

Is there a difference between _successful_ and _successfully complete_? If so and we desire _successfully complete_ perhaps it can be _after the plan has successfully completed_.

Author: Hrmmm interesting question ...

Originally, I thought the main difference (I see) is that we use the term "complete" frequently in documentation.  BUT we do say "A plan was succesful". So yes, I think that is a valid way of describing it!

Then, is "plan is succesful" less or more readable than "plan is successfully complete". I'd say it's the same and more succinct. 

And finally, should it be in present or past tense. The sentence is talking about  "post-plan executes after ..." so it should be past tense.

i.e.`The post-plan phase executes any configured run tasks after a successful plan`,

Is that better?

Author: Oh, and I'll apply the same wording to the post-apply section too.

Reviewer 2: In documentation we always prefer present tense and active voice for clarity! 

I'm not sure if this applies here, but the main difference between "complete plan" and "successful plan", is the the case where a plan errors and cannot finish successfully. You could argue that a failed plan is still "complete", but not "successful". 

How would you feel about a suggestion like below? Echoing the language from the post-apply phase:
```suggestion
The post-plan phase only occurs if you configure [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) on a workspace to begin after Terraform successfully completes a plan operation.
```

Author: That sounds good to me Rose.  Thankyou!

@dylanegan you ok if I use that wording too?

Reviewer: 👍 thank you so much @rkoron007. 🙇‍♂️

Author: Done.

## The Post-Apply Stage

> The post-apply phase only occurs if the workspace has [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) configured to begin after Terraform successfully completes an apply operation. During this phase, HCP Terraform sends information about the run to the configured external system and waits for a `passed` or `failed` response. However, unlike other stages in the run task process, a failed outcome will not halt the run since HCP Terraform has already provisioned the infrastructure.

Reviewer 2: Teeny suggestions for active voice and present tense:
```suggestion
The post-apply phase only occurs if you configure [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) on a workspace to begin after Terraform completes an apply operation. During this phase, HCP Terraform sends information about the run to the configured external system and waits for a `passed` or `failed` response. However, unlike other stages in the run task process, a failed outcome does not halt the run since HCP Terraform has already provisioned the infrastructure.
```

Author: Done
